pandas/tests/frame/test_reshape.py

```diff
@@ -282,6 +283,22 @@ def test_unstack_fill_frame_categorical(self):
                              index=list('xyz'))
         assert_frame_equal(result, expected)
 
+    def test_unstack_preserve_dtypes(self):
+        # Checks fix for #11847
```

Comment:
can you write a 1-liner that this is checking for a list of unstacks is the same as as repeated unstacking.